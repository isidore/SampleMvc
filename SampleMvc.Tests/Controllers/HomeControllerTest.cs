using ApprovalTests.Asp;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace SampleMvc.Tests.Controllers
{
	[TestClass]
	[UseReporter(typeof(DiffReporter))]
	public class HomeControllerTest
	{
		[TestMethod]
		public void About()
		{
			AspApprovals.VerifyUrl("http://localhost:50011/Home/About");
		}
	}
}