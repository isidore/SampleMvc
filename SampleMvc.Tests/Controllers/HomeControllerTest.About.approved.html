<!DOCTYPE html>
<html>
<head><base href="http://localhost:50011/Home">
    <title>About Us</title>
    <link href="/Content/Site.css" rel="stylesheet" type="text/css" />
    <script src="/Scripts/jquery-1.5.1.min.js" type="text/javascript"></script>
</head>
<body>
    <div class="page">
        <div id="header">
            <div id="title">
                <h1>My MVC Application</h1>
            </div>
            <div id="logindisplay">
                    [ <a href="/Account/LogOn">Log On</a> ]

            </div>
            <div id="menucontainer">
                <ul id="menu">
                    <li><a href="/">Home</a></li>
                    <li><a href="/Home/About">About</a></li>
                </ul>
            </div>
        </div>
        <div id="main">
            
<h2>About</h2>
<p>
	This is my lovely static page.
</p>

        </div>
        <div id="footer">
        </div>
    </div>
</body>
</html>
